ansible/latest-packages.yml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: cluster_latest_packages
+  become: yes
+  tasks:
+
+    - name: install latest packages
+      yum:
+        name: "*"
+        state: latest
+
+    - name: check if reboot is required
+      command: needs-restarting -r
+      failed_when: false
+      register: reboot_required
+      changed_when: false
+
+    - when: reboot_required.rc != 0
```

Comment:
FYI We have a reboot_and_wait role in this repo.

Reply:
Thanks, I'll include that in.